store/models/common_test.go

```diff
@@ -249,7 +249,34 @@ func TestWebURL_String_HasNilURL(t *testing.T) {
 	assert.Equal(t, "", w.String())
 }
 
-func TestTimeDurationFromNow(t *testing.T) {
+func TestTime_UnmarshalJSON(t *testing.T) {
+	tests := []struct {
+		name    string
+		input   string
+		want    time.Time
+		errored bool
+	}{
+		{"unix string", `"1529445491"`, time.Unix(1529445491, 0).UTC(), false},
+		{"unix int", `1529445491`, time.Unix(1529445491, 0).UTC(), false},
+		{"iso8601", `"2018-06-19T22:17:19Z"`, time.Unix(1529446639, 0).UTC(), false},
```

Comment:
How flexible is this? iso8601 technically means: `2018` is valid, so is `2018-06-19` etc. I wouldn't expect exhaustive testing of this, but it might be nice to add another like "iso8601 date" vs "iso8601 timestamp"?

Reply:
👍I would assume there would be an issue interpreting `2018` as seconds since the unix epoch.

Reply:
My aim was to test the formats we care about, and beyond that the extra variations in formats are  all tested in github.com/araddon/dateparse. I'm open to writing a few more tests or clarifying in the test name.